scripts/calibre-web-nginx.conf

```diff
@@ -1,7 +1,46 @@
 # RELATED: https://github.com/janeczku/calibre-web/wiki/Setup-Reverse-Proxy
 
+# 2023-12-04: This is part of an (interim?) workaround so that playback of
+# larger videos actually works.  Far better would be a proper MM (memory
+# management) solution that respects Calibre-Web's existing username/password
+# security model.  Can anybody suggest concrete ideas to make that happen?
+#
+# The failure of large videos to play is summarized here:
+# https://github.com/iiab/calibre-web/issues/37
+#
+# This (perhaps imperfect) proposed solution is emerging here:
+# https://github.com/iiab/calibre-web/pull/50
+# https://github.com/iiab/calibre-web/pull/51
+# https://github.com/iiab/calibre-web/issues/53
+# https://github.com/iiab/iiab/pull/3676
+# https://github.com/iiab/calibre-web/pull/55
+# https://github.com/iiab/calibre-web/pull/56
+# https://github.com/iiab/calibre-web/pull/57
+
 # http://box/library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>
 location /library/calibre-web/ {
     alias /library/www/html/calibre-web/;
-    fancyindex on;    # autoindex on;
+    # fancyindex on;
+    # autoindex on;
+    
```

Comment:
"Lint" to a void excess/inconsistent whitespace:

```suggestion

```

Reply:
CLARIF: Please remove the 4 blanks spaces on Line 25, to help be more consistent with similar lines, e.g. Lines 33 and 40.

(A linting tool can do this fully automatically, if possible in future!)